nflows/transforms/splines/rational_quadratic.py

```diff
@@ -34,16 +33,18 @@ def unconstrained_rational_quadratic_spline(
         constant = np.log(np.exp(1 - min_derivative) - 1)
         unnormalized_derivatives[..., 0] = constant
         unnormalized_derivatives[..., -1] = constant
-
-        outputs[outside_interval_mask] = inputs[outside_interval_mask]
-        logabsdet[outside_interval_mask] = 0
+        
+        outputs = torch.where(outside_interval_mask, inputs, outputs)
+        logabsdet = torch.where(outside_interval_mask, torch.zeros_like(logabsdet), logabsdet)
```

Comment:
Actually, does the original line even have an effect? We're assigning zeros to what is already initialized to zeros.